Cargo.toml

```diff
@@ -248,6 +250,7 @@ k256 = { version = "0.13.3", default-features = false }
 elliptic-curve = { version = "0.13.8", default-features = false }
 ecdsa = { version = "0.16.9", default-features = false }
 num-bigint = { version = "0.4.6", default-features = false }
+num-bigint-dig = { version = "0.8.4", default-features = false }
```

Comment:
we are no longer using `num-bigint-dig` as its not as maintained as `num-bigint`. please switch over to using `num-bigint` and remove this dep